App.js

```diff
@@ -12,6 +12,7 @@ const sentryApi = secrets[namespace].sentryApi;
 if (sentryApi?.dsn) {
   Sentry.init({
     dsn: sentryApi.dsn,
+    enableInExpoDevelopment: true,
```

Comment:
what is the reason for the decision to report dev errors to sentry?

Reply:
please check the line two lines after this one, where a comment describes the situation

Reply:
I did rebase and deleted `enableInExpoDevelopment`.

https://github.com/smart-village-solutions/smart-village-app-app/commit/2c91de40ca84dc76fb38888ce7606499e6ea5f5e